src/compositor/lipstickcompositor.cpp

```diff
@@ -71,7 +71,10 @@ LipstickCompositor::LipstickCompositor()
     , m_mceNameOwner(new QMceNameOwner(this))
     , m_sessionActivationTries(0)
 {
-    setColor(Qt::black);
+    QGuiApplication::platformNativeInterface()->nativeResourceForIntegration("DisplayOn");
+    setColor(Qt::magenta);
```

Comment:
This could be behind an environment variable so that it's easier to use for testing purposes.

Reply:
Done like that now